pkgs/applications/misc/collision/gi-crystal/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, crystal
+, libadwaita
+, gobject-introspection
+}:
+crystal.buildCrystalPackage rec {
+  pname = "gi-crystal";
+  version = "0.16.0";
+
+  src = fetchFromGitHub {
+    owner = "hugopl";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
This is a constant back and forth, depending on the reviewer…

Some sub-ecosystems already have agreed on enforcing `refs/tags/*`, others are doing it randomly.

In my opinion we should agree on a treewide policy to keep (and add where its missing!) `refs/tags/` ASAP. But this discussion is not to be held in this PR.

Reply:
I looked at the code in other packages and saw that quite a few people do fetching this way, so I thought I could do that too.

Reply:
@NobbZ I wasn't aware of that! Good to know.

If there are inconsistencies, please ignore this feedback and keep it as it was, it's fine.